include/z64save.h

```diff
@@ -4,6 +4,28 @@
 #include "ultra64.h"
 #include "z64math.h"
 
+typedef enum {
+    /*  0 */ HUD_NO_CHANGE,
+    /*  1 */ HUD_HIDE_ALL,
+    /*  2 */ HUD_HIDE_ALL_ALT, // Identical to HUD_HIDE_ALL
+    /*  3 */ HUD_SHOW_HEARTS_FORCE, // See below
+    /*  4 */ HUD_SHOW_A,
+    /*  5 */ HUD_SHOW_A_HEARTS_MAGIC_FORCE, // See below
+    /*  6 */ HUD_SHOW_A_HEARTS_MAGIC_MINIMAP_FORCE, // See below
+    /*  7 */ HUD_HIDE_MINIMAP_BY_BTN_STATUS, // Only raises button alphas if not disabled
+    /*  8 */ HUD_SHOW_B,
+    /*  9 */ HUD_SHOW_HEARTS_MAGIC,
+    /* 10 */ HUD_SHOW_B_ALT, // Identical to HUD_SHOW_B
+    /* 11 */ HUD_SHOW_HEARTS,
+    /* 12 */ HUD_SHOW_A_B_MINIMAP,
+    /* 13 */ HUD_SHOW_HEARTS_MAGIC_FORCE, // See below
+    /* 50 */ HUD_SHOW_ALL = 50, // Only raises button alphas if not disabled
+    /* 52 */ HUD_HIDE_ALL_INSTANT = 52
+} HudVisibilityMode;
+
+// `_FORCE` means that this request will respond to `forceRisingButtonAlphas`.
+// If set, the buttons will also raise alphas but will also account for disabled buttons
```

Comment:
This comment could maybe do with being above the enum, that way it may be less likely to get lost/overlooked?

Reply:
And as a doxygen comment? does that show up, I forgot